GCP_VM_Delete.yml

```diff
@@ -5,8 +5,6 @@
   gather_facts: no
   vars:
     gcp_project: ceenter
-    scopes:
-      - https://www.googleapis.com/auth/compute
```

Comment:
hardcoded url - it should be in variable

Reply:
+1

Reply:
I am removing it in this commit, it is default value, so no need to keeping it around.